obs-package-status/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+while true; do
+  ./status_check.rb
```

Comment:
If you make this `$1` or (`$@`?) it should be whatever we pass as `CMD`.

Reply:
yup, if we were building a base image, then I would agree with that, but because it's specific for a job, I don't mind to have it there, and don't have it as argument to ```docker run```. I don't have a strong opinion here. How do you want to have it?

* ```CMD``` being passed in the ```Dockerfile```
* Pass it as parameter to ```docker run```
* keep as it is

Reply:
CMD being passed in the Dockerfile.

Reply:
done, can you please check it again @hennevogel ?